src/features/applicationView/offerCard/sendInvoiceForm/SendInvoiceForm.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import * as Yup from 'yup';
+import { useTranslation } from 'react-i18next';
+import { TextInput } from 'hds-react';
+import { Form, Formik } from 'formik';
+import { TFunction } from 'i18next';
+
+import styles from './sendInvoiceForm.module.scss';
+import FormHeader from '../../../../common/formHeader/FormHeader';
+import Button from '../../../../common/button/Button';
+import Text from '../../../../common/text/Text';
+
+type FormValues = {
+  dueDate: string;
+};
+
+export type SendInvoiceFormProps = {
+  email: string | null;
+  isSubmitting: boolean;
+  onCancel: () => void;
+  onSubmit: (data: FormValues) => void;
+};
+
+const getValidationSchema = (t: TFunction) =>
```

Comment:
You could just initialize the validationSchema here:

```
const validationSchema = Yup.object<FormValues>().shape({
    dueDate: Yup.date().required(t('forms.common.errors.required')),
  });
```

And you that variables directly in the formik.

Reply:
example: https://formik.org/docs/guides/validation#validationschema

Reply:
`t` is not available outside of the `SendInvoiceForm` scope

Reply:
Oh, my bad, completely missed that. In palvelutarjotin this is handled luttle bit differently, only keys are given to schema. 

With useMemo this repeated schema call could be avoided but I don't if it is worth it.